frontend/src/metabase/collections/containers/CollectionSidebar/CollectionSidebar.jsx

```diff
@@ -79,90 +72,46 @@ class CollectionSidebar extends React.Component {
     return (
       <React.Fragment>
         <ToggleMobileSidebarIcon onClick={handleToggleMobileSidebar} />
+
         <Collection.Loader id="root">
-          {({ collection: root }) => (
-            <Box mb={1} mt={2}>
-              <CollectionDropTarget collection={root}>
-                {({ highlighted, hovered }) => (
-                  <CollectionLink
-                    to={Urls.collection({ id: "root" })}
-                    selected={isRoot}
-                    highlighted={highlighted}
-                    hovered={hovered}
-                  >
-                    {t`Our analytics`}
-                  </CollectionLink>
-                )}
-              </CollectionDropTarget>
-            </Box>
-          )}
+          {({ collection: root }) => <Header isRoot={isRoot} root={root} />}
         </Collection.Loader>
 
-        <Box pb={4}>
-          <CollectionsList
-            openCollections={this.state.openCollections}
-            onClose={this.onClose}
-            onOpen={this.onOpen}
-            collections={list}
-            filter={nonPersonalOrArchivedCollection}
-            currentCollection={collectionId}
-          />
-
-          <Box>
-            <CollectionsList
-              openCollections={this.state.openCollections}
-              onClose={this.onClose}
-              onOpen={this.onOpen}
-              collections={currentUserPersonalCollections(list, currentUser.id)}
-              initialIcon="person"
-              filter={this.filterPersonalCollections}
-              currentCollection={collectionId}
-            />
-          </Box>
-        </Box>
-
-        <Box className="mt-auto" pb={2} pl={SIDEBAR_SPACER * 2}>
-          {currentUser.is_superuser && (
-            <Link
-              my={2}
-              to={Urls.collection({ id: "users" })}
-              className="flex align-center text-bold text-light text-brand-hover"
-            >
-              <Icon name="group" mr={1} />
-              {t`Other users' personal collections`}
-            </Link>
-          )}
-          <Link
-            to={`/archive`}
-            className="flex align-center text-bold text-light text-brand-hover"
-          >
-            <Icon name="view_archive" mr={1} />
-            {t`View archive`}
-          </Link>
-        </Box>
+        <Collections
+          collectionId={collectionId}
+          currentUserId={currentUser.id}
+          list={list}
+          onClose={this.onClose}
+          onOpen={this.onOpen}
+          openCollections={this.state.openCollections}
+        />
+
+        <Footer isSuperUser={currentUser.is_superuser} />
```

Comment:
From what I see, we usually use `isAdmin`. `is_superuser` seems to be a legacy of some ancient times when Metabase was written in [Django](https://docs.djangoproject.com/en/3.2/ref/contrib/auth/#django.contrib.auth.models.User.is_superuser)